go.mod

```diff
@@ -19,10 +19,11 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-chi/chi v4.0.2+incompatible
 	github.com/go-chi/cors v1.0.0
+	github.com/go-git/go-git/v5 v5.1.0
```

Comment:
Upgrading go.mod in nuclio is delicate. can you use the current versions to reach the pr goal?